### PR TITLE
feat: add Nolus to chain list

### PR DESCRIPTION
**Pull request by Lockwarr**
Chain registry - https://github.com/cosmos/chain-registry/tree/master/nolus

### CODE REVIEW

File: cosmos/nolus.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/nolus/nolus.png"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "NLS",
+      "coinMinimalDenom": "unls",
+      "coinDecimals": 6,
+      "coinGeckoId": "",
```
Comment (HeesungB) [inline cosmos/nolus.json:39]: @Lockwarr Could you remove `coinGeckoId` field?

**Comment by Lockwarr** [comment]
@HeesungB should I include nolus in `isNativeSupported` in src/index.ts to fix the failing build?

**Comment by Lockwarr** [comment]
@HeesungB locally, the build fails with error `invalid image url`.. should the `nolus.png` image be submitted on a separate PR?

**Comment by HeesungB** [comment]
@Lockwarr Hello.
Your chain id is `pirin-1`, so Your chain identifier is `pirin`.
Your image directory name will be `pirin`, not nolus!
Could you change name?

**Comment by HeesungB** [comment]
@Lockwarr Could you remove below fields?
```
 "walletUrl": "https://wallet.keplr.app/chains/nolus",
 "walletUrlForStaking": "https://wallet.keplr.app/chains/nolus",
  ```

**Comment by Lockwarr** [comment]
@HeesungB I have removed them, I also brought back coingeckoID since we have it now

**Pull request merged**